test/integration/spec/connect.js

```diff
@@ -787,6 +799,30 @@ describe('connect', function() {
             assert(maxBitrate ? bLinePattern.test(section) : !bLinePattern.test(section));
           });
         });
+
+        // TODO: Remove firefox check once this firefox bug is fixed
+        // https://bugzilla.mozilla.org/show_bug.cgi?id=1688342
+        if (!isFirefox) {
+          it(`should ${maxBitrates[kind] ? '' : 'not '}limit the ${kind} bitrate`, () => {
+            const averageBitrate = kind === 'audio' ? averageAudioBitrate : averageVideoBitrate;
+            const minBitrate = kind === 'audio' ? minAudioBitrate : minVideoBitrate;
+            if (maxBitrates[kind]) {
+              const hasLessBitrate = averageBitrate <= maxBitrates[kind];
+              if (!hasLessBitrate) {
+                // Log values if failed
+                console.log(`maxBitrate exceeded. desired: ${maxBitrates[kind]}, actual: ${averageBitrate}`);
+              }
+              assert(hasLessBitrate);
+            } else {
+              const hasUnlimitedBitrate = averageBitrate > minBitrate;
+              if (!hasUnlimitedBitrate) {
+                // Log values if failed
+                console.log(`Bitrate is unexpectedly low. ${maxBitrates[kind]}, actual: ${averageBitrate}`);
+              }
+              assert(hasUnlimitedBitrate);
```

Comment:
This should also log the bitrate in case the assertion fails:
```suggestion
              assert(hasUnlimitedBitrate, `Bitrate is unexpectedly low. ${maxBitrates[kind]}, actual: ${averageBitrate}`);
```